src/background.js

```diff
@@ -49,20 +51,24 @@ if ('commands' in chrome) {
 
 // *************************************************************************
 {
-  const onInstall = ({reason}) => {
+  const onInstall = ({
+    reason
```

Comment:
This robotic expansion of literals doesn't make the code more readable. On the contrary, it's less readable in such cases as this one where only one parameter is used.